mathics/builtin/drawing/README.md

```diff
@@ -0,0 +1,53 @@
+This README concerns testing implemented in `test_plot_detail.py`,
```

Comment:
Please incorporate this into the developer documents.

Reply:
> > Please incorporate this into the developer documents.
> 
> I don't at the moment know how to do that. Can you point me to the specific repo artifact I need to change, and suggest a specific location in the document for it?

https://github.com/Mathics3/Mathics3-development-guide See section https://mathics-development-guide.readthedocs.io/en/latest/extending/developing-code/testing.html

Reply:
Here https://github.com/Mathics3/Mathics3-development-guide/pull/45#issue-3765346280  I have copied the text of this readme. Later, we can find a better place, or a better organization for this document.

Reply:
In any case, I think by now this is not a bad place to keep this information. Eventually, this file can be reduced to a link to the development guide.

Reply:
> In any case, I think by now this is not a bad place to keep this information. Eventually, this file can be reduced to a link to the development guide.

The problem with keeping this information has been noted many times in database design: redundancy can lead to duplicate updating and possibly inconsistency. 

Please let's add the link first. Otherwise, what is likely to happen is that we'll go to the next major incomplete overhaul, losing interest in finishing the first one. 

It is not that hard to add a link now; it's not that much effort.

Reply:
@bdlucas1 Here is the README.md